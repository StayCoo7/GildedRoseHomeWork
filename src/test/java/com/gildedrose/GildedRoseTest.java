package com.gildedrose;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class GildedRoseTest {

    @Test
    void foo() {
        Item[] items = new Item[]{new Item("hhhh", -2, -3), new Item("Sulfuras, Hand of Ragnaros", 1, -1),
            new Item("Backstage passes to a TAFKAL80ETC concert", -2, 3), new Item("Aged Brie", 1, 3),
            new Item("Backstage passes to a TAFKAL80ETC concert", 20, 49), new Item("Aged Brie", -1, 50),
            new Item("Backstage passes to a TAFKAL80ETC concert", 1, 49), new Item("Aged Brie", -1, 4),
            new Item("Backstage passes to a TAFKAL80ETC concert", 1, 66), new Item("hhh", 1, 4),
            new Item("hhhh", -2, 3), new Item("Sulfuras, Hand of Ragnaros", -2, -3)};
        GildedRose app = new GildedRose(items);
        app.updateQuality();
        Approvals.verifyAll("", items);

        //assertEquals("fixme", app.items[0].name);
    }

}
