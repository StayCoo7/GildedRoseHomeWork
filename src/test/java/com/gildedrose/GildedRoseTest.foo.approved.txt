[00] = hhhh, -3, -3
[01] = Sulfuras, Hand of Ragnaros, 1, -1
[02] = Backstage passes to a TAFKAL80ETC concert, -3, 0
[03] = Aged Brie, 0, 4
[04] = Backstage passes to a TAFKAL80ETC concert, 19, 50
[05] = Aged Brie, -2, 50
[06] = Backstage passes to a TAFKAL80ETC concert, 0, 50
[07] = Aged Brie, -2, 6
[08] = Backstage passes to a TAFKAL80ETC concert, 0, 66
[09] = hhh, 0, 3
[10] = hhhh, -3, 1
[11] = Sulfuras, Hand of Ragnaros, -2, -3
